src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -76,7 +78,7 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         Person personToEdit = lastShownList.get(index.getZeroBased());
-        Person editedPerson = createEditedPerson(personToEdit, editPersonDescriptor);
+        Person editedPerson = createEditedPerson((Student) personToEdit, editPersonDescriptor);
```

Comment:
Currently it's necessary because the method takes in `Person`. But once we implement different edit commands for student and teacher it will no longer be needed. @nniiggeell

Reply:
Yup sadly it is, because I need to change createEditedPerson to take in a Student. Unless maybe we have a class that maybe extends from EditCommand, perhaps EditStudentCommand